homeassistant/components/unifiprotect/number.py

```diff
@@ -124,7 +124,7 @@ def _get_chime_duration(obj: Camera) -> int:
         name="Infrared custom lux trigger",
         icon="mdi:white-balance-sunny",
         entity_category=EntityCategory.CONFIG,
-        ufp_min=1,
+        ufp_min=0,
```

Comment:
Please do this change in a separate PR

Reply:
Any time a PR title has `and` in the title, its a good hint that we should follow this policy https://developers.home-assistant.io/docs/creating_component_code_review?_highlight=smalle#5-make-your-pull-request-as-small-as-possible and split the PR